bin/npm/install.js

```diff
@@ -0,0 +1,112 @@
+var Chalk = require('chalk');
+var Bluebird = require('bluebird');
+var Cli = require('structured-cli');
+var Semver = require('semver');
+var Superagent = require('superagent');
+var Request = require('sandboxjs/lib/issueRequest');
+var npmRunner = require('../../lib/npmRunner');
+
+module.exports = Cli.createCommand('install', {
+    description: 'Installs a package',
+    handler: handleInstall,
+    options: {
+      // provided as convienience
+      save: {
+        action: 'storeTrue'
+      }
+    },
+    params: {
+      varargs: {
+          description: 'Libraries to install',
+          type: 'string',
+          required: true,
+          nargs: '*'
+      }
+    }
+});
+
+function getAvailableModules () {
+  var LIST_MODULES_URL = 'https://webtask.it.auth0.com/api/run/wt-tehsis-gmail_com-1?key=eyJhbGciOiJIUzI1NiIsImtpZCI6IjIifQ.eyJqdGkiOiJmZGZiOWU2MjQ0YjQ0YWYyYjc2YzAwNGU1NjgwOGIxNCIsImlhdCI6MTQzMDMyNjc4MiwiY2EiOlsiZDQ3ZDNiMzRkMmI3NGEwZDljYzgwOTg3OGQ3MWQ4Y2QiXSwiZGQiOjAsInVybCI6Imh0dHA6Ly90ZWhzaXMuZ2l0aHViLmlvL3dlYnRhc2tpby1jYW5pcmVxdWlyZS90YXNrcy9saXN0X21vZHVsZXMuanMiLCJ0ZW4iOiIvXnd0LXRlaHNpcy1nbWFpbF9jb20tWzAtMV0kLyJ9.MJqAB9mgs57tQTWtRuZRj6NCbzXxZcXCASYGISk3Q6c';
+  var request = Superagent.get(LIST_MODULES_URL);
+  return Request(request)
+          .get('body')
+          .then(function (body) {
+            return body.modules;
+          });
+}
+
+
+// Command handler
+function handleInstall (args) {
+  var requestedLibs = args.varargs.filter(isNotOption).map(toLibObject);
+  var passThroughArgs = args.varargs.filter(isOption);
+
+  if (args.save) {
+    passThroughArgs.unshift('--save');
```

Comment:
Do you think that maybe we should use `--save-exact` to prevent the locally installed version from drifting away from the pinned version on the platform? Not sure how we would wrangle the data to make this possible.